### PR TITLE
Update skype-room-systems-v2.md

### PR DESCRIPTION
**Pull request by msbemba**
Added GCC and GCC high Dod Sku's

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/room-systems/skype-room-systems-v2.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Microsoft has and dedicated SKU for licensing meetings and calling on a per-devi
 
 
 > [!NOTE]
-> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.  
+> If you are currently using E1, E3, E4, E5, GCC or GCC High and DoD SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/skype-room-systems-v2.md]:
```suggestion
> If you are currently using E1, E3, E4, E5, GCC or GCC High, and DoD SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.  
```

Comment (msbemba) [reply Teams/room-systems/skype-room-systems-v2.md]: @JohanFreelancer9 "Gcc High and Dod" is the name of the license. Let me know if this still needs a change.

Comment (JohanFreelancer9) [reply Teams/room-systems/skype-room-systems-v2.md]: @msbemba Thank you for correcting me. No need to change.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin Any updates on this? Thanks

**Comment by scanum** [comment]
@LanaChin @waseemhashem @jambirk PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by jambirk** [comment]
Waseemhashem was mentioned in error, approval needed is from @sohailta

**Comment by scanum** [comment]
@sohailta PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by jambirk** [comment]
according to Sohail GCC or GCC High and DoD SKUs are definitely not supported. Since they aren't supported, let's close the PR.

**Pull request closed without merging**